### DISCUSSION TITLE
Issues in lecture transcription

### DISCUSSION THREAD

**Discussion by Askanders** [openai/whisper#1394] [Ideas]
Hi, I'd just like to give some feedback.
I'm using whisper to transcribe medical lectures, I do it with colab with the large model and italian language. The audio quality of the files is good and I've tested the program with other more general purpose audios with great results.
What happens though with medical lectures in particular is that the AI makes many mistakes with technical terms, lots of them and without consistency, the same word gets interpreted in different ways based on the sentence.
Often I'm able to correct the mistakes giving the text to ChatGPT but this usually results in a loss of information.

Anyway, great work, this new technologies are really changing the way I'm approaching my studies, I can use my time focusing on comprehension and revision with the AI helping me removing the tedious, mechanical tasks that used to take so much time.

Thank you for your efforts, hope this little feedback will be useful.

**Comment by phineas-pta** [comment]
it's a problem of training data which didn't cover good enough academic subjects

if you want to improve accuracy, you have to fine tune the model on lectures audio

**Reply by Emichel2019** [reply]
Has there been any new models that have been created and already fine tuned with medical and healthcare related audio

**Reply by phineas-pta** [reply]
no idea
